Dockerfile

```diff
@@ -1,3 +1,17 @@
+# (Keep the version in sync with the node install below)
+FROM node:20-slim as frontend
+
+# Make build & post-install scripts behave as if we were in a CI environment (e.g. for logging verbosity purposes).
+ARG CI=true
+
+# Install front-end dependencies.
+COPY package.json package-lock.json tsconfig.json webpack.config.js tailwind.config.js  ./
```

Comment:
Question: Do these files need loading as part of `npm ci`? The fewer files used here, the better caching will be.

Reply:
I think so - I copied this over from kit.